tests/redux/reducers/award/awardReducer-test.jsx

```diff
@@ -0,0 +1,420 @@
+                    },
+                    {
+                        id: 36791,
+                        type: 'D',
+                        type_description: 'Definitive Contract',
+                        action_date: '10/15/2015',
+                        action_type: null,
+                        federal_action_obligation: '$0',
+                        modification_number: '911',
+                        description: 'MANAGEMENT AND OPERATION OF THE OAK RIDGE NATIONAL LABORATORY - REVISE APPENDIX E TO INCORPORATE RCN OR-62'
+                    },
+                    {
+                        id: 34869,
+                        type: 'D',
+                        type_description: 'Definitive Contract',
+                        action_date: '10/30/2014',
+                        action_type: null,
+                        federal_action_obligation: '-$14,274',
+                        modification_number: '860',
+                        description: 'MANAGEMENT AND OPERATION OF THE OAK RIDGE NATIONAL LABORATORY - RECOVERY TAS::89 0227::TAS - DEOBLIGATE SC ARRA FUNDING FROM PROJECT ENTITLED "NUCLEAR PHYSICS - NUCLEAR SCIENCE WORKFORCE"'
+                    }
+                ]
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.transactions).toEqual(action.transactions);
+        });
+    });
+
+    describe('SET_TRANSACTIONS_META', () => {
+        it('should save the returned award to redux state', () => {
+            const action = {
+                type: 'SET_TRANSACTIONS_META',
+                count: 13,
+                page: 1,
+                totalPages: 2
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.transactionMeta).toEqual(action.transactionMeta);
+        });
+    });
+
+    describe('SET_TRANSACTION_SORT', () => {
+        it('should set the sorting column and direction', () => {
+            const action = {
+                type: 'SET_TRANSACTION_SORT',
+                direction: 'desc',
+                field: 'modification_number'
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.transactionSort).toEqual(action.transactionSort);
+        });
+    });
+
+    describe('UPDATE_TXN_RENDER_HASH', () => {
+        it('should provide a unique hash id for the transaction render', () => {
+            const action = {
+                type: 'UPDATE_TXN_RENDER_HASH',
+                renderHash: _.uniqueId()
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.renderHash).not.toEqual(action.renderHash);
```

Comment:
To validate that the hash has changed, we need to execute the Redux function multiple times (twice) and test that the hash changed between those times.